index.js

```diff
@@ -32,6 +34,10 @@ function gulpAmdCheck(options) {
         file.contents = new Buffer(output.optimizedContent);
       }
 
+      if(options.errorOnUnusedDependencies && output.results.length && output.results[0].unusedDependencies.length) {

```

Comment:
No, a file can contain more than 1 module. `output.results` is the collection of those processed modules.